src/components/Inputs/TextArea/TextArea.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import TextArea from '.';
+
+const title = 'SimplenightUI/Inputs/';
+
+export default {
+  title: `${title}Text Area`,
+  component: TextArea,
+} as ComponentMeta<typeof TextArea>;
+
+const Template: ComponentStory<typeof TextArea> = (args) => (
+  <TextArea {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  placeholder: 'Placeholder',
+};
```

Comment:
Would be great to add more examples, with idle, error, and success, just to have them